syntax = "proto3";

package Packet;

message GameResultsPacket {
	string murderer = 1;
	string voting_result = 2;
	repeated uint32 voting_result_points = 3;
	string winner = 4;
}